### DISCUSSION TITLE
how to setup on muti devices to improve availability?

### DISCUSSION THREAD

**Discussion by MisteryMonster** [filebrowser/filebrowser#1280] [Q&A] [answered]
I want to share the same folder (using syncthing) in different devices (both windows, desktop and laptop), and if one turning offline that i can use another to keep service running in same site. 
So I use load balancing mode in my [frp](https://github.com/fatedier/frp#load-balancing) proxy, setting filebrowser auth mode in `Proxy Header`, but it will cause 403 error ramdonly, I think it's cause by sending the same header `auth:` token in different backend that doesn't recorded, even I turn `noauth` both but no help. I really don't know much about it, is the auth "problem" or I can do it with different methods?

**Comment by o1egl** [comment, marked as answer]
Try to upgrade to the latest version

**Reply by MisteryMonster** [reply]
403 error either, maybe is the URL still attached `?auth=` ？
In console:
```
GEThttps://domain/api/preview/thumb/something/329.JPG?auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7ImlkIjoyLCJsb2NhbGUiOiJ6aC1jbiIsInZpZXdNb2RlIjoibW9zYWljIiwic2luZ2xlQ2xpY2siOmZhbHNlLCJwZXJtIjp7ImFkbWluIjpmYWxzZSwiZXhlY3V0ZSI6ZmFsc2UsImNyZWF0ZSI6ZmFsc2UsInJlbmFtZSI6ZmFsc2UsIm1vZGlmeSI6ZmFsc2UsImRlbGV0ZSI6ZmFsc2UsInNoYXJlIjp0cnVlLCJkb3dubG9hZCI6dHJ1ZX0sImNvbW1hbmRzIjpbXSwibG9ja1Bhc3N3b3JkIjpmYWxzZSwiaGlkZURvdGZpbGVzIjpmYWxzZX0sImV4cCI6MTYxNzI4NzU0NCwiaWF0IjoxNjE3MjgwMzQ0LCJpc3MiOiJGaWxlIEJyb3dzZXIifQ.xAF3STaFgICvWo4ppU1moyj9LZp339XxjPEUkuqaz5o&inline=true&k=1595834319782
[HTTP/2 403 Forbidden 483ms]

	
GET
	https://domain/api/preview/thumb/something/329.JPG?auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7ImlkIjoyLCJsb2NhbGUiOiJ6aC1jbiIsInZpZXdNb2RlIjoibW9zYWljIiwic2luZ2xlQ2xpY2siOmZhbHNlLCJwZXJtIjp7ImFkbWluIjpmYWxzZSwiZXhlY3V0ZSI6ZmFsc2UsImNyZWF0ZSI6ZmFsc2UsInJlbmFtZSI6ZmFsc2UsIm1vZGlmeSI6ZmFsc2UsImRlbGV0ZSI6ZmFsc2UsInNoYXJlIjp0cnVlLCJkb3dubG9hZCI6dHJ1ZX0sImNvbW1hbmRzIjpbXSwibG9ja1Bhc3N3b3JkIjpmYWxzZSwiaGlkZURvdGZpbGVzIjpmYWxzZX0sImV4cCI6MTYxNzI4NzU0NCwiaWF0IjoxNjE3MjgwMzQ0LCJpc3MiOiJGaWxlIEJyb3dzZXIifQ.xAF3STaFgICvWo4ppU1moyj9LZp339XxjPEUkuqaz5o&inline=true&k=1595834319782
Status403
Forbidden
VersionHTTP/2
Transferred211 B (14 B size)
Referrer Policystrict-origin-when-cross-origin

    	
    content-length
    	14
    content-type
    	text/plain; charset=utf-8
    date
    	Thu, 01 Apr 2021 12:32:34 GMT
    server
    	nginx/1.16.0
    x-content-type-options
    	nosniff
    X-Firefox-Spdy
    	h2
    	
    Accept
    	image/webp,*/*
    Accept-Encoding
    	gzip, deflate, br
    Accept-Language
    	en-US,en;q=0.5
    Authorization
    	Basic 5a6/566h6YOoOjEyMzQ1Ng==
    Cache-Control
    	max-age=0
    Connection
    	keep-alive
    DNT
    	1
    Host
    	domain
    Referer
    	https://domain/files/something/
    TE
    	Trailers
    User-Agent
    	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:87.0) Gecko/20100101 Firefox/87.0
```

**Reply by ramiresviana** [reply]
To have an authentication token working on different instances of Filebrowser, both will need to have the same security key. The key is stored in the database, you can clone the database file or use cli to import and export the configuration.

**Reply by MisteryMonster** [reply]
Thank you so much! I copy the database it finally works! But before that I have tried CLI export and import config ( It just ignored the key string ) and modified the database with [boltbrowser](https://github.com/ShoshinNikita/boltBrowser) to make the key string same but it cause 500 internal server error.
